android/CouchbaseLite/src/main/java/com/couchbase/lite/ConsoleLogger.java

```diff
@@ -30,13 +31,13 @@
  */
 public final class ConsoleLogger implements Logger {
     private LogLevel _level = LogLevel.WARNING;
-    private ArrayList<LogDomain> _domains = new ArrayList<>();
+    private EnumSet<LogDomain> _domains = EnumSet.noneOf(LogDomain.class);
```

Comment:
I think you could initialize the _domains here to LogDomain.ALL then we don't need to do anything in the ConsoleLogger() constructor.